packages/eslint/src/generators/lint-project/lint-project.ts

```diff
@@ -199,6 +204,15 @@ function createEsLintConfiguration(
   const pathToRootConfig = extendedRootConfig
     ? `${offsetFromRoot(projectConfig.root)}${extendedRootConfig}`
     : undefined;
+
+  if (extendedRootConfig) {
+    // We do not want to mix the formats
+    options.eslintConfigFormat = tree
+      .read(extendedRootConfig, 'utf-8')
+      .includes('export default')
```

Comment:
The file may not be formatted correctly, especially now that we're making Prettier optional. We should parse out the AST and check that default export exists.